test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/SigningUtilityTests.cs

```diff
@@ -109,22 +109,21 @@ public void Verify_WhenChainBuildingFails_Throws()
                 Assert.Equal("Certificate chain validation failed.", exception.Message);
 
                 Assert.Equal(1, logger.Errors);
-                Assert.Equal(1, logger.Warnings);
 
-                if (RuntimeEnvironmentHelper.IsWindows)
+                if (!RuntimeEnvironmentHelper.IsMacOSX && RuntimeEnvironmentHelper.IsLinux)
                 {
-                    AssertNotTimeValid(logger.LogMessages, LogLevel.Error);
-                    SigningTestUtility.AssertUntrustedRoot(logger.LogMessages, LogLevel.Warning);
-                }
-                else if (RuntimeEnvironmentHelper.IsMacOSX)
+                    Assert.Equal(2, logger.Warnings);
+                }else
                 {
-                    AssertExpiredCertificate(logger.LogMessages, LogLevel.Error);
-                    SigningTestUtility.AssertUntrustedRoot(logger.LogMessages, LogLevel.Warning); 
+                    Assert.Equal(1, logger.Warnings);
                 }
-                else
+
+                SigningTestUtility.AssertNotTimeValid(logger.LogMessages, LogLevel.Error);
+                SigningTestUtility.AssertUntrustedRoot(logger.LogMessages, LogLevel.Warning);
+
+                if (!RuntimeEnvironmentHelper.IsMacOSX && RuntimeEnvironmentHelper.IsLinux)
                 {
-                    AssertPartialChain(logger.LogMessages, LogLevel.Error);
-                    SigningTestUtility.AssertOfflineRevocation(logger.LogMessages, LogLevel.Warning);
+                    SigningTestUtility.AssertRevocationStatusUnknown(logger.LogMessages, LogLevel.Warning);
```

Comment:
Move this statement inside [this block](https://github.com/NuGet/NuGet.Client/pull/3130/files#diff-15324c04203c0921ea53645aa5eb5f66R113-R116).